messages/subjects.yaml

```diff
@@ -140,11 +141,16 @@ target_tcpa_s:                          keelson.TimestampedFloat
 target_bcr_m:                           keelson.TimestampedFloat
 target_bct_s:                           keelson.TimestampedFloat
 
-device_uptime_duration:                 keelson.TimestampDuration
+device_uptime_duration:                 keelson.TimestampedDuration
 
 radio_channel_ppm_pct:                  keelson.TimestampedFloat
 radio_rssi:                             keelson.TimestampedFloat
 
+time_source_gnss:                       keelson.TimestampedTimestamp  # Global Navigation Satellite System
+time_source_ptp:                        keelson.TimestampedTimestamp  # Precision Time Protocol 
+time_source_rtc:                        keelson.TimestampedTimestamp  # Real-Time Clock
+time_source_system:                     keelson.TimestampedTimestamp  # System clock of the computer
```

Comment:
Arent these just different sources of time? So they are essentially different `source_id`s in the key? Why do we need different subject for them?